pkg/ddevapp/config.go

```diff
@@ -365,12 +365,23 @@ func (app *DdevApp) docrootPrompt() error {
 	output.UserOut.Printf("\nThe docroot is the directory from which your site is served. This is a relative path from your application root (%s)", app.AppRoot)
 	output.UserOut.Println("You may leave this value blank if your site files are in the application root")
 	var docrootPrompt = "Docroot Location"
-	if app.Docroot != "" {
-		docrootPrompt = fmt.Sprintf("%s (%s)", docrootPrompt, app.Docroot)
+	// Provide use the app.Docroot as the default docroot option.
+	var defaultDocroot = app.Docroot
+	if defaultDocroot == "" {
+		for _, docroot := range []string{"web", "docroot", "htdocs", "_www", "public"} {
```

Comment:
Let's define these outside of inline code please. They should be the same in tests as in the definition code. Just a global array definition should be fine.

Reply:
Please make this work with web/public (normal composer location for TYPO3) as well.

Reply:
Shouldn't it be looking in the approot/project root first? Or am I missing something here?

Reply:
Perhaps we should....

* web/public (TYPO3, otherwise default composer template wins)
* web
* docroot
* htdocs
* public
* _www
* _default to approot_

Reply:
Probably if approot is already defined we have to stick with it. So it should be the default? But then we have the ambiguity of "" being the docroot vs "" being unset.

Reply:
I guess I meant default being "" which is approot? or is approot == docroot?

All I meant was "Default the last option to being the current directory." Which maybe hitting enter does that" Maybe just say `Docroot location: (current directory)`

Reply:
Approot or project root is the base of the repo (or where .ddev has been created). Docroot is where index.php is.  Sometimes they are the same.

Reply:
Cool. So I think we cycle through the list. If nothing provided as default the prompt should be `Docroot location: (current directory)` so you know you can hit enter and go. That's the current behavior, but without the hint, correct?

Reply:
👍